app/errors/invalid_suri_syntax.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+##
+# this allows us to re-raise certain StandardErrors from moab-versioning in a more
+# specifically rescue-able way
+class InvalidSuriSyntax < StandardError
+end
```

Comment:
ideally, i'd have put this in the moab-versioning gem, made the raised error more specific from there, bumped the version, and used the newly cut gem.  since this is really a prescat specific thing.  but i was trying to be a bit expedient.  totally happy to do that as follow on work if time permits.